clang/lib/Sema/SemaTemplateDeductionGuide.cpp

```diff
@@ -317,7 +317,8 @@ struct ConvertConstructorToDeductionGuideTransform {
     }
 
     if (NestedPattern)
-      OuterInstantiationArgs = SemaRef.getTemplateInstantiationArgs(Template);
+      OuterInstantiationArgs = SemaRef.getTemplateInstantiationArgs(
+          /*D=*/nullptr, Template->getDeclContext());
```

Comment:
What's happening here? Why do we want to pass in a null `Decl` and its DeclContext here?

Reply:
This is an artifact from when I was figuring out the implementation... I'll revert this change.